src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App" />
+  );
+}
+
```

Comment:
- [ ] Also add a CreateNewBook component it should render a form with inputs for the title and the author of the book, it should also have a button to submit, it won't actually submit but it's just a placeholder for now.